base.js

```diff
@@ -0,0 +1,35 @@
+
+function renderErrors(errors) {
+	for (var i=0; i < errors.length; i++) {
+		$("#errors").append("<div> &raquo; " + errors[i] + "</div>").show();
```

Comment:
Unnecessary hiding/showing of #errors container - content is appended later, so no need for using CSS and JS for error message toggling.